### PR TITLE
direct change for variables

### PR DESCRIPTION
**Pull request by Bettman66**
direct change for variables of the new script language.
You can create the data point in the script:
```
>D
p:sollwert=0
>S
if mqttc>0 {
=>publish stat/%topic%/RESULT {"Script":">sollwert=%1sollwert%"}
}
>W
subs:{m} %1sollwert% °C
```
also on the console of Tasmota:
`Script>sollwert=25`

The data point is displayed in the Sonoff adapter with `Script_sollwert` and can then be edited directly.

### CODE REVIEW

**Comment by GermanBluefox** [comment]
If this feature is configurable in sonoff, so you must describe it in readme. Because if the user will use other name than "Scrip_" it will not work

File: lib/server.js [hunk 1]
```diff
@@ -1151,10 +1159,13 @@ function MQTTServer(adapter) {
             if (str.startsWith('{"POWER":"ON"}')) return;
             if (str.startsWith('{"POWER":"OFF"}')) return;
 
-
             if (parts[0] === 'stat') {
                 try {
-                    checkData(client, packet.topic, NO_PREFIX, JSON.parse(val));
+                    checkData(client, packet.topic, NO_PREFIX, JSON.parse(val));			
+                    if (val.includes('Script')) {
```
Comment (GermanBluefox) [inline lib/server.js:1165]: Should be here `val.includes('Script_')` too as above?

Comment (Bettman66) [reply lib/server.js:1165]: No, Tasmota sends a json e.g. {"script": ">soll=20"}. I turn it into an object script_soll with the value 20. When sending script_soll with the value 30 to Tasmota it has to be converted back to {"script": ">soll=30"}.
The values can be changed directly in the new Tasmota scripting language. To remain downward compatible, the Script object with the value ">soll=20" is also created. You could also write ">=30" in the Script object, but this is complicated.

Comment (Bettman66) [reply lib/server.js:1165]: Look here:
[https://forum.iobroker.net/topic/26237/beca-thermostate-einbinden-firmware-mod/48](url)

**Comment by Bettman66** [comment]
> If this feature is configurable in sonoff, so you must describe it in readme. Because if the user will use other name than "Scrip_" it will not work

{"Script":">variable=0"} is the command in Tasmota to change variables in the script. If you want I can write a howto in the readme or my own script readme with a specific application.

**Comment by GermanBluefox** [comment]
> If you want I can write a howto in the readme or my own script readme with a specific application.

I think, it is required to make this feature understandable for end user and for me :)

**Pull request closed without merging**